java/test/src/main/resources/test_cross_language_invocation.py

```diff
@@ -9,14 +9,22 @@
 @ray.remote
 def py_func(value):
     assert isinstance(value, bytes)
-    return b"Response from Python: " + value
+    f = ray.java_function("org.ray.api.test.CrossLanguageInvocationTest",
+                          "BytesEcho")
+    r = f.remote(value)
+    return b"Response from Python: " + ray.get(r)
```

Comment:
Maybe we can make the words more explicit:
`The return object generated from the python task py_func.`

So that wherever we meet this message, we can know the details from it rather than tracing the code to debug.